unittests/abi_tests.cpp

```diff
@@ -551,8 +551,9 @@ BOOST_AUTO_TEST_CASE(general)
 { try {
 
    auto abi = eosio_contract_abi(fc::json::from_string(my_abi).as<abi_def>());
+   auto abi2 = abi;
 
-   abi_serializer abis(abi, abi_serializer::create_yield_function( max_serialization_time ));
+   abi_serializer abis(std::move(abi), abi_serializer::create_yield_function( max_serialization_time ));
```

Comment:
Instead of introducing abi2, why not:

`abi_serializer abis(abi_def(abi), abi_serializer::create_yield_function( max_serialization_time ));`